### PR TITLE
Merge two schema, concatenating array items of same type

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: server/utils/schema-merge.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		throw Error(`Can't merge ${type(a[key])} with ${type(b[key])} for key [${key}]`);
+	});
+
+	return a;
+};
+
+module.exports = (...objects) => {
+	return objects
+		.reduce(
+			(acc, current) => mergeObjects(acc, current),
```
Comment (adambraimbridge) [inline server/utils/schema-merge.js]: Acc == Accrued?

**Comment by adambraimbridge** [comment]
Very well done 👍

File: server/utils/schema-merge.js (hunk 1 above)
Comment (georgecrawford) [inline server/utils/schema-merge.js]: "The reduce() method applies a function against an accumulator and each value of the array (from left-to-right) to reduce it to a single value." Accumulator is the thing which collects all the data from the other things.

**Comment by commuterjoy** [comment]
+1

File: server/utils/schema-merge.js (hunk 1 above)
Comment (adambraimbridge) [inline server/utils/schema-merge.js]: Aha! I always called it `returnArray` or `returnObject` and used coffee to wash the bad taste out of my mouth. `acc` (or `accumulator`) is nicer. :)

Comment (georgecrawford) [inline server/utils/schema-merge.js]: `accumulator` would have been clearer ;)

**Pull request merged**